en/editor/assets/index.md

```diff
@@ -111,6 +111,28 @@ The search function is a combined function that can limit the search type and sp
 - Shortcut key `Esc` to cancel rename
 - In addition, the initial name of the Typescript asset will be treated as its `className`, and the `className` cannot be repeated.
 
+### Exporting Assets Package
+
+Select the resource to be exported and right click on it, then select **Export Resource Package**. The **Export Resource** panel will automatically list the currently selected resource and its dependencies. If you don't need to export dependent resources, you can uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.
```

Comment:
Select the resource to be exported and right click on it,
-->
Select the resource to be exported, right-click on it,

Reply:
If you don't need to export dependent resources, you can uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.
-->
To not export the dependent resources, uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.